src/microbe_stage/MicrobeHUD.cs

```diff
@@ -170,6 +170,12 @@ public class MicrobeHUD : Node
     private GridContainer environmentPanelBarContainer;
     private Panel compoundsPanel;
 
+    // Store these statefully for after player death
+    private float hp;
```

Comment:
hp and atpAmount shouldn't need saving, right? As if the player is dead they can't have any health or ATP.

Reply:
The fact that a dead microbe doesn't still contain ATP is a current game rule, not a absolute truth, so I'd be reluctant to assume that here just in case. In general it felt neater to treat such closely tied variables in the same way, but I have no objections if you want them as strictly method vars.

Reply:
Even if a dead microbe would contain ATP, what use is it to show the player?
I guess the player could see if they died due to running out of ATP...
But the hp variable here definitely should go as it serves no purpose.